packages/puppeteer-core/src/common/ElementHandle.ts

```diff
@@ -45,9 +46,11 @@ const applyOffsetsToQuad = (
   offsetX: number,
   offsetY: number
 ) => {
+  assert(quad.length === 4);
```

Comment:
Provide a correct error message, so it's easier to find if it fails.

Reply:
This will never fail.